### PR TITLE
Task8

### PR DESCRIPTION
**Pull request by NDragneelL9**
1. Created graphics page
2. Added list of transactions to graphics page
2. Added feature to choose date range for transactions

### CODE REVIEW

File: app/controllers/pages_controller.rb [hunk 1]
```diff
@@ -1,4 +1,19 @@
 # Controller to pages entities
 class PagesController < ApplicationController
   def index; end
+
+  def graphics
+    time_now = Time.zone.now
+    @date_from = time_now.to_date.beginning_of_month
+    @date_to = time_now.to_date.next_day
+    transactions_date_range if params[:search].present?
+    @personality = UserPersonality.find(params[:user_personality_id])
```
Comment (e-popolamov) [inline app/controllers/pages_controller.rb:10]: image
I think it is wrong behaviour

**Review by e-popolamov — COMMENTED** [review]
When fix this problem please re-request me

File: app/controllers/pages_controller.rb (hunk 1 above)
Comment (e-popolamov) [reply app/controllers/pages_controller.rb:10]: In other places all good and i think tomorrow u can pass this

Comment (NDragneelL9) [reply app/controllers/pages_controller.rb:10]: Fixed

**Review by e-popolamov — APPROVED** [review]

**Comment by e-popolamov** [comment]
Congratulations! You finished all tasks!

**Pull request merged**